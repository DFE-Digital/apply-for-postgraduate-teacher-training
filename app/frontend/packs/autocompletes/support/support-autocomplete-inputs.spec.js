import {supportAutocompleteInputs} from "./support-autocomplete-inputs";

describe("supportAutocompleteInputs", () => {
  it("should return supportAutocompleteInputs", () => {
    expect(supportAutocompleteInputs).toMatchSnapshot();
  });
});
