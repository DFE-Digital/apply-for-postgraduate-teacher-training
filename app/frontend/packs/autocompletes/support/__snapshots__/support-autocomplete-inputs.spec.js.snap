// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`supportAutocompleteInputs should return supportAutocompleteInputs 1`] = `
Array [
  Object {
    "autocompleteId": "api-token-autocomplete",
    "inputIds": Array [
      "#vendor-api-token-provider-id-field.govuk-select",
    ],
  },
  Object {
    "autocompleteId": "country-autocomplete",
    "inputIds": Array [
      "#support-interface-application-forms-edit-address-details-form-country-field",
      "#support-interface-application-forms-edit-address-details-form-country-field-error",
    ],
  },
]
`;
